datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```diff
@@ -0,0 +1,92 @@
+- name: host
+  required: true
+  description: JMX hostname to connect to.
+  value:
+    example: localhost
+    type: string
+
+- name: port
+  required: true
+  description: JMX port to connect to.
+  value:
+    example: 8686
```

Comment:
Is this standard? If not, let's not put example

Reply:
`8686` seems used quite often for `jmx`, not sure it's standard, but it's quite common/typical:

- https://docs.oracle.com/cd/E19528-01/819-4892/ganjx/index.html
- https://www.ibm.com/support/knowledgecenter/en/SSHS8R_8.0.0/com.ibm.worklight.installconfig.doc/install_config/t_optional_config_app_server_tomcat.html
- https://thwack.solarwinds.com/t5/SAM-Documents/Understanding-Java-Remote-JMX-Initial-Setup-Config-Overview/ta-p/529969